CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ----------
 
+[0.27.2] - 2017-03-10
```

Comment:
@pomegranited Should be 0.28 - the schema is major.minor.patch where

* major - for backward-incompatible changes
* minor - for new backward-compatible features 
* patch - for bugfixes.

Checkout semver schema (in the comment above) - makes a lot of sense when you read it.